flink-cdc-cli/src/test/resources/log4j2-test.properties

```diff
@@ -13,7 +13,7 @@
 
 # Set root logger level to OFF to not flood build logs
 # set manually to INFO for debugging purposes
-rootLogger.level = INFO
+rootLogger.level = OFF
```

Comment:
How about changing the level to ERROR so that we can see some exception message in log?

Reply:
Changed to `ERROR`, but should be noted that this might log some internal "errors" that are actually expected.

(For example, an SchemaEvolveException might be printed out if we're intentionally testing a schema evolution failure case, while the test case itself might succeed.)